File: Swift/프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.swift
import Foundation

func solution(_ my_string:String) -> [Int] {
    var answer = [Int](repeating: 0, count: 52)
    
    for i in my_string.unicodeScalars {
        var unicode = Int(i.value)
        
        if unicode >= 65 && unicode <= 90 {
            answer[unicode - 65] += 1
        } else {
            answer[unicode - 71] += 1
        }
    }

    return answer
}

File: Python3/프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    answer = [0 for _ in range(52)]
    
    for char in my_string:
        char_int = ord(char)
        if 65 <= char_int <= 90:
            answer[char_int-65] += 1
        else:
            answer[char_int-71] += 1
    
    return answer